projects/module-2/fitlit-part-two-agile.md

```diff
@@ -218,44 +218,63 @@ Once your usability test is complete, incorporate any useful and interesting fee
-- 2: Project has sporadic use of tests at multiple levels. The application contains numerous holes in testing and some tests do not reflect changes made to implementation. No use of `beforeEach` to DRY up tests. Tests must be passing to be considered.
-- 1: Tests have not been updated to reflect changes made in refactor. There is not enough test coverage, and some tests might be failing.
+<section class="answer">
+### Testing
+- Application has a robust and thorough test suite.
+- Test suite covers all scenarios/outcomes, including happy and sad paths.
+- Test suite is organized.
+- Tests use smaller, sample data files as input rather than the large, original data files.
+- `beforeEach` hook is used to DRY up test files
+- There are no failing/pending tests upon submission
+</section>
 
+<section class="answer">
 ### Accessibility
 
-- 4: Has an audit score of 100% and has gone above and beyond accessibility requirements (see extensions).
-- 3: All accessibility areas have been considered and tested. An accessibility audit with a score of at least 95%.
-- 2: One accessibility area has not been tested or considered. This may be: an accessibility audit with a score of 94% or lower, errors with color contrast when checking via a colorblind extension, app is not tab-able, large use of non-semantic HTML.
-- 1: Two or more accessibility areas have not been tested or considered. This may be: an accessibility audit with a score of 80% or lower, errors with color contrast when checking via a colorblind extension, app is not tab-able, large use of non-semantic HTML.
+- The Lighthouse audit tool was used to test and improve accessibility.
+- Application uses semantic HTML.
+- Users can have a seamless experience using only a keyboard to navigate the application (no mouse).
+- Users who have color-blindness can utilize this application without issue.
+- ARIA attributes are used on interactive elements if needed - and *only* if needed.
+* Wow option: design is responsive across small, medium and large breakpoints
+</section>
 
+<section class="answer">
 ### JavaScript Refactoring
+- Code has been refactored into logical components each with a clean, single responsibility
+- Array prototype methods are used to iterate instead of for loops
+- All DOM manipulation is held in the `domUpdates.js` file.  No DOM manipulation occurs outside of this file.
+- Application handles all scenarios for error handling
+- `fetch` is effectively implemented for GET and POST
+- DOM is updated based on the results of network requests
+- Demonstrates efforts towards making functions pure when possible. *Note: Purity is not possible for every function in a FE application. Strive for it only when it makes sense.*
+- WOW option: Effectively implements one or more closure throughout project.  *Note: See Closures lesson on M2 lesson page as a resource.*
```

Comment:
Potentially we could **bolden** Wow options just to help with readability and highlight which notes are extra?  I know it's nit picky, but it would help my eyes when skimming through the rubric!

Reply:
Yep! I have another update to these rubrics coming and I will add that in - thanks Travis!